docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -29,14 +29,16 @@ To use the Data Quality dashboard, you need at least the following {ref}/securit
 [discrete]
 [[data-quality-dash-check-indices]]
 == Check indices
-When you open the dashboard, data does not appear until you select indices to check.
+When the dashboard gets opened for the first time in a project, data does not appear until you (or a teammate) select indices to check. 
 
 * *Check multiple indices*: To check all indices in the current data view, click *Check all* at the top of the dashboard. A progress indicator will appear.
 
 IMPORTANT: To customize which indices are checked when you click *Check all*, {security-guide}/data-views-in-sec.html[change the current data view].
 
 * *Check a single index*: To check a single index, expand it using the arrow on the left. Checking a single index is faster than checking all indices.
 
+Once checked, an index's data quality information persists indefinitely. You can see when it was last checked, and recheck it at any time.
```

Comment:
I'm not entirely clear what "information" means in the first sentence. If you're referring to the results of the data quality check, maybe swap out "information" with "results" or "details" instead.

```suggestion
Once checked, an index's data quality results persist indefinitely. You can see when it was last checked, and recheck it at any time.
```